android/build-envoy.sh

```diff
@@ -16,7 +16,7 @@ export CMAKE_HOME=$ANDROID_SDK_ROOT/cmake/${CMAKE_VERSION}/bin
 export PATH=$PATH:$ANDROID_SDK_ROOT/platform-tools:$CMAKE_HOME
 export sdkmanager=$ANDROID_SDK_ROOT/tools/bin/sdkmanager
 
-set -e
+set -euo pipefail
```

Comment:
This change causes a failure on the step where the SDK licenses are being accepted if they've already been accepted. I assume it returns an error code in that case and causes a pipe failure. I'll fix this